### PR TITLE
reduces cas cost for ammo overall and ds surgery table cost

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
dropship table costed the pilots their entire point supply for the round, reduced it from 300 to 25.
Reduced ammo costs overall by half or 1/3rdish on average. GAU reduced massively.

## Why It's Good For The Game
So you can actually get ammo and not be fudged because someone joined as medical doctor

## Changelog
:cl:
balance: reduced cas ammo cost overall and dropship surgery table
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -105. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by neohenry** [comment]
Finally I can rush Alamo surgery table as CMO.

**Comment by Chaznoodles** [comment]
If you can be bothered making map edits, I'd swap out the MEDEVAC system that spawns in storage for the operating table. It's never used anyway.

File: code/game/objects/structures/dropship_equipment.dm [hunk 1]
```diff
@@ -807,7 +807,7 @@
 	equip_category = DROPSHIP_CREW_WEAPON
 	icon = 'icons/obj/surgery.dmi'
 	icon_state = "table2-idle"
-	point_cost = 300
+	point_cost = 25
```
Comment (Hughgent) [inline code/game/objects/structures/dropship_equipment.dm:810]: This is excessive

**Review by Hughgent — APPROVED** [review]
variable changes

File: code/game/objects/structures/dropship_equipment.dm (hunk 1 above)
Comment (TiviPlus) [reply code/game/objects/structures/dropship_equipment.dm:810]: Make it 100

**Review by TiviPlus — APPROVED** [review]

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-117). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**